dispatcher.go

```diff
@@ -54,185 +33,11 @@ type Dispatcher interface {
 	OnDeviceEvent(*device.Event)
 }
 
-// dispatcher is the internal Dispatcher implementation
-type dispatcher struct {
```

Comment:
@johnabass I moved this to it's file and renamed it to `eventDispatcher` since it's taking care of dispatching events to Caduceus and other endpoints